<!DOCTYPE html>
<html lang="en">
<head>
@require("../partials/head.html")
</head>
<body>
@require("../partials/navigation.html")
<div class="container">

  <h1 id="user-guide">User guide</h1>
  <p>
    PrankWeb is a web-based application that allows to predict and visualize
    protein-ligand binding sites. Furthermore, it allows to compare the
    location of predicted pockets with highly conserved areas as well as actual
    ligand binding sites. All one needs to use PrankWeb is a device with a
    web-browser that supports WebGL.
  </p>

  <h2 id="specify-what-protein-to-analyze">
    Specify what protein to analyze
  </h2>
  <p>
    There are two options to analyze a protein:
  </p>
  <ul>
    <li>
      PrankWeb can download it automatically from PDB database provided the
      protein identification code.
    </li>
    <li>
      A protein PDB file can be uploaded from user's computer. For description
      of PDB format, please see its
      <a href="http://www.wwpdb.org/documentation/file-format" target="_blank">
        official documentation
      </a>.
    </li>
  </ul>

  <h3>Homology</h3>
  <p>
    Besides selecting what protein to analyze, one can also specify whether
    evolutionary conservation should be included in the prediction model by
    checking the <b>Run conservation analysis</b> checkbox. Note that
    calculating conservation score for user-defined protein file can
    significantly increase the time of analysis unless you specify its
    PDB identification code or upload multiplesequence alignments for homology
    calculation.
  </p>
  <p>
    There are three ways how to calculate the conservation score for the
    protein:
  </p>
  <ul>
    <li>
      You can specify custom alignment file, using which PrankWeb will
      calculate the conservation score for your protein.
    </li>
    <li>
      If you know the PDB code of you protein, you can specify in the the
      textbox and PrankWeb will calculate the homology from the alignment
      in
      <a href="http://swift.cmbi.ru.nl/gv/hssp/" target="_blank">
        HSSP database
      </a>.
    </li>
    <li>
      Fallback method: If any of the previous methods fails or neither pdb
      code nor the alignment file is specified, PrankWeb will calculate the
      conservation score automatically using alignment of similar proteins.
    </li>
  </ul>

  <h2 id="visualization">Visualization</h2>
  <p>
    Once the protein visualization is loaded, three main panels appear:
    sequence visualization, structural visualization and the pocket panel.
  </p>
  <img src="./assets/images/help-00.png"
       style="width: 100%"
       class="img-responsive center-block"
    />
  <h3 id="structural-visualization-1">Structural visualization</h3>
  <p>
    The largest panel contains the three-dimensional visualization of the
    protein.
  </p>
  <ul>
    <li>
      By default, the protein surface is displayed, and individual pocket
      areas are highlighted with different colors. Ligands (if available)
      are displayed with yellow.
    </li>
    <li>
      If conservation is available, the protein atoms is colored with 11
      shades of gray according to the conservation score of each residue.
      Darker color depicts higher conservation score.
    </li>
    <li>
      To switch between the views of the protein, use
      <b>Peptide visualisation</b> and <b>Pockets visualisation</b>
      selections at top of the pocket panel on the right.
    </li>
  </ul>

  <h4 id="controls">Controls</h4>
  <p>
    The molecule can be rotated by moving mouse while holding left mouse button.
    On a touch device, just slide your finger. To zoom in or out, move your
    mouse while holding the right mouse button or use the pinch gesture on
    a touch display. In order to move the protein, do the same, but this time
    hold the wheel button. Lastly, for slabbing the protein, scroll the mouse
    wheel or use the three finger gesture.
  </p>
  <p>Using the buttons in the top-right corner, one can:</p>
  <ul>
    <li>Display a help window.</li>
    <li>
      Setup the scene such as the visualization background or the field of
      view.
    </li>
    <li>Create a snapshot of current visualization.</li>
    <li>Toggle advanced control panels.</li>
    <li>Toggle full-screen mode.</li>
  </ul>
  <p>
    By toggling the advanced control panel, one has full control over the
    content of the visualization using LiteMol. For more help with LiteMol,
    please visit its
    <a
      href="https://webchem.ncbr.muni.cz/Wiki/LiteMol:UserManual"
      target="_blank"
    >
      wiki page
    </a>.
  </p>

  <h3 id="sequence-visualization-1">Sequence visualization</h3>
  <p>The panel above protein 3D visualization displays protein sequence.</p>
  <ul>
    <li>
      All chains are concatenated and visualized at once. Chains can be
      identified by regions marked on the main sequence.
    </li>
    <li>
      Colored rectangles depict areas with predicted pockets and real binding
      areas (if available).
    </li>
    <li>
      Real binding sites are residues within 4 &#8491; from any ligand
      atom.
    </li>
    <li>
      If available, conservation scores are portrayed using a bar chart.
    </li>
  </ul>
  <p>
    As one hovers over the sequence with mouse, the residues are highlighted
    in the 3D visualization. This feature allows to analyze the protein both
    from the structural and sequential point of view. By default, the sequence
    view is zoomed out so that the whole protein is displayed. You can use the
    trackbar control to zoom in, or select the area with mouse and zoom to the
    selection. A snapshot of the sequence can be captured and exported to SVG
    (Scalable Vector Graphics) file using the rightmost button.
  </p>

  <h2 id="pocket-panel">Pocket panel</h2>
  <p>
    The right panel contains several control buttons and a list of predicted
    pockets. Use the control buttons to download PrankWeb report, hide the
    sequence view or switch between the structural views. In the pocket list,
    pocket name, rank, size and average conservation score (if available) is
    displayed for each pocket. Each pocket also contains several buttons
    related to visibility of the pockets in the visualizations.
  </p>

  <h2 id="contact-us">Contact us</h2>
  <p>
    Something is not working or are you missing certain functionality/feature?
    Please let us know by creating a
    <a
      href="https://github.com/cusbg/prankweb/issues/new/choose"
      target="_blank"
    >
      GitHub issue
    </a>.
    Alternatively, if you prefer to use email, please feel free to reach us at
    david.hoksza&nbsp;(at)&nbsp;mff.cuni.cz.
  </p>

  @require("../partials/footer.html")

</div>
</body>
</html>